### DISCUSSION TITLE
Bad colors while using neotree

### DISCUSSION THREAD

**Discussion by HenningCode** [nvim-neo-tree/neo-tree.nvim#1454] [General]
Hey Guys,

I just installed neo-tree and set a basic keymap to toggle neo-tree, but the colors are off. Can someone tell me why that is? 
![image](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/57907680/63de901e-bff4-400c-aff8-7b892d678705)

This is my neo-tree config: 
```lua
return {
    "nvim-neo-tree/neo-tree.nvim",
    branch = "v3.x",
    dependencies = {
      "nvim-lua/plenary.nvim",
      "nvim-tree/nvim-web-devicons", 
      "MunifTanjim/nui.nvim",
    },
    config = function()
        vim.keymap.set('n', '<C-n>', ':Neotree filesystem toggle right<CR>', {})
    end
}
```

and this is my colorscheme setup:
```lua 
return {
    "sainnhe/gruvbox-material",
    name = "gruvbox-material",
    lazy = false,
    priority = 1000,

    config = function()
        vim.g.gruvbox_material_background = 'soft'
        vim.cmd("colorscheme gruvbox-material")
    end
}
```

I am pretty new to neovim to excuse me if I made any stupid mistakes.

**Comment by pynappo** [comment]
[This seems intentional on gruvbox's side.](https://github.com/sainnhe/gruvbox-material/blob/146f40fd42cbef30fed69b4ef51329aeeaceb909/colors/gruvbox-material.vim#L1863) it uses a darker bg than normal.

If you want to change it then you're free to modify the NeoTreeNormal background yourself
